### ISSUE TITLE
Can Dockerfile use a later version of the node engine?

### ISSUE THREAD

**Issue by Zloka** [NickNaso/ghostscript4js#63]
The `Dockerfile` includes the following comment:
`# Official Node.js 8.x lts image based on Debian stretch (has ghostscript 9.20 in its archive)`

Is it not in the archive of later versions? A bit new to this, so how could I go about verifying this myself? :)

**Comment by Qwerios** [comment]
I've just been able to at least get through npm install for the current `node:22` (which is bookworm) with the following: `ENV GS4JS_HOME=/usr/lib/aarch64-linux-gnu`

Except from `Dockerfile` looks like this:

```
FROM node:22

# Add required system dependencies
#
RUN apt-get update
RUN apt-get install -y ghostscript libgs-dev

# Ghostscrip4js needs to be informed of the location of the libgs.so file
#
ENV GS4JS_HOME=/usr/lib/aarch64-linux-gnu

# COPY your sources and package.json stuff here

RUN npm install

# ENTRYPOINT and such here
```

**Comment by Qwerios** [comment]
The aarch64 path appears to be specific when building on an M-series CPU.
When I pushed the build to a hosted linux docker builder (GCE in this case) I had to set the ENV variable to `/usr/lib/x86_64-linux-gnu` as per the docs

**Comment by NickNaso** [comment]
> The aarch64 path appears to be specific when building on an M-series CPU. When I pushed the build to a hosted linux docker builder (GCE in this case) I had to set the ENV variable to `/usr/lib/x86_64-linux-gnu` as per the docs

Yes, in this case you don't need to set `GS4JS_HOME` env variable because the default is `/usr/lib/x86_64-linux-gnu`.
